addons/mail/static/src/utils/messaging_component.js

```diff
@@ -33,10 +35,31 @@ export function registerMessagingComponent(ComponentClass, { propsCompareDepth =
             // useModels as if they were part of the OWL rendering itself.
             useModels();
             useShouldUpdateBasedOnProps({ propsCompareDepth });
+            onPatched(() => {
+                const className = this.className.trim()
+                if (className.length === 0) {
+                    return;
+                }
+                this.el.classList.add(className);
+            });
```

Comment:
mounted too actually, so `onUpdate`, also className needs to split space and add each word separately apparently

Reply:
Not sure whether current implementation of `useUpdate` allows 2 uses of this hook by a same component.
There's extra code for listening on models that is not useful here but maybe problematic if a component has several occurrences of `useUpdate`?

Reply:
I don't think there is any restriction to the number of `useUpdate` that can be called?

If we later change `className` to be a field and `useUpdate` is not used, it will fail to observe it.